action/action.go

```diff
@@ -251,7 +251,22 @@ func injectParameters(action string, c *claim.Claim, env, files map[string]strin
 			}
 			continue
 		}
-		value := fmt.Sprintf("%v", rawval)
+
+		contents, err := json.Marshal(rawval)
```

Comment:
I wonder if the following code could be simpler, with something like this:
```go
value, ok = rawval.(string)
if !ok {
    // rawval is not a string, we marshal it as JSON
    contents, err := json.Marshal(rawval)
    if err != nil {
	return err
    }
    value := string(contents)
}
```
WDYT?
Sorry about this back and fourth 😅

Reply:
The reason we marshal to strings here only to throw the work away was I worried about knowing (for sure) that the golang value would only marshal to a string if its type was already a string. Since this is a library that could be used by CLI implementations other than Duffle, I feel like other marshal-to-string-but-aren't-literally-strings types could end up here and it seems friendlier to give a clear error message rather than ending up with bonus quotes in the values.

Reply:
Right that makes sense!